src/summary/documentation/documentation.component.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import { match } from "react-router";
+import { getCurrentPatient } from "@openmrs/esm-api";
+import SummarySectionHeader from "../summary-section-header.component";
+import SummarySectionCards from "../cards/summary-section-cards.component";
+import VitalsCard from "./vitals-card.component";
+// import IdentifiersCard from "./identifiers-card.component";
```

Comment:
Please do not push commented code. Additionally, please do not include someone else's code in your pull requests. Fatma created the IdentifiersCard and ContactsCard. You should not have them in your branch.

You should wait until her code is on the master branch before incorporating it into your codebase. This is called [trunk based development](https://trunkbaseddevelopment.com/) and is the standard for open source software. It looks like you might have done this because you wanted to use the HorizontalLabelValue that she created in #2. What you should have done instead is ask her to maybe separate that component out into its own PR that is easily mergeable into the master branch. Then once it is merged, use it. You should not ever be copy pasting someone else's branch of code, nor should you be basing your pull requests off of anything except the master branch.